.github/ISSUE_TEMPLATE/release-check-list.md

```diff
@@ -0,0 +1,26 @@
+---
+name: New Release Check List
+about: v<TARGET_RELEASE>
+title: "v<TARGET_RELEASE Check List"
+---
+
+## v<TARGET_RELEASE>
+
+* [ ] - Get the latest ocicrypt-rs used as part of image-rs
+* [ ] - Get the image-rs, using the latest ocicrypt-rs, as part of attestastion-agent
+- [ ] - Get the latest attestation-agent, using the latest image-rs, as part of the Enclave CC
+- [ ] - Get the latest attestation-agent, using the latest image-rs, as part of the Kata Containers
```

Comment:
So we pull in image-rs and the attestation-agent independently in Kata Containers:
- image-rs is pulled in via the kata-agent's Cargo.toml (twice, once for non-s390x and the other for s390x)
- attestation-agent via the versions.yaml file
I think we want to reflect that in this bullet?